chart/flux/templates/deployment.yaml

```diff
@@ -210,7 +210,7 @@ spec:
           - --git-set-author={{ .Values.git.setAuthor }}
           - --git-poll-interval={{ .Values.git.pollInterval }}
           - --git-timeout={{ .Values.git.timeout }}
-          - --sync-interval={{ .Values.git.pollInterval }}
+          - --sync-interval={{ .Values.sync.interval }}
```

Comment:
I favor this change, but it breaks backwards compatibility. A safer solution would be to keep setting `--sync-interval` to `.Values.git.pollInterval` **unless** `.Values.sync.interval` is provided.

Reply:
@stefanprodan thoughts on this? I can make this change if you prefer that approach to the one I took. I don't have a strong preference either way.

Reply:
@2opremio I've reworked this so it will default to `.Values.git.pollInterval` if `.Values.sync.interval` isn't provided.

Reply:
If all is good, what does the release cadence for helm charts look like?

Reply:
There will be a release soon (CC @stefanprodan )